firebase-firestore/firebase-firestore.gradle

```diff
@@ -83,6 +83,7 @@ android {
         sourceCompatibility JavaVersion.VERSION_1_8
         targetCompatibility JavaVersion.VERSION_1_8
     }
+    testOptions.unitTests.includeAndroidResources = true
```

Comment:
This is technically tangential to your PR, right?  Please confirm you're including it intentionally.

Reply:
Also, please confirm that including this flag now does not generate a warning.

Reply:
> This is technically tangential to your PR, right? Please confirm you're including it intentionally.

Mostly tangential, though it causes the issue to manifest itself. (Also required by Ashwin's changes.)

> Also, please confirm that including this flag now does not generate a warning.

This does not seem to introduce any new warnings.